Sprintfinity3902/Maps/DefaultMap.cs

```diff
@@ -73,27 +79,35 @@ public void Setup(Game1 gameInstance)
             itemIndex = KeyboardManager.Instance.CreateNewDeltaKeys(Keys.U, Keys.I);
             NPCIndex = KeyboardManager.Instance.CreateNewDeltaKeys(Keys.O, Keys.P);
 
-        }
+            KeyboardManager.Instance.RegisterCommand(new SetDamageLinkCommand(gameInstance), Keys.E);
```

Comment:
I very intentionally left these out of the default map last night. The default map becomes too cluttered and its focus is lost. I like the idea of using default map to generate all of the things that correlate to the specific room the player is presently in.

I do not like the idea of default map being a place to hold miscellaneous game related things and artificially reduce the amount of code in game1. This doesn't add to code simplicity.

Reply:
This goes for both the boomerang items that link uses (he has those in every room, the room shouldn't generate it) as well as the keyboard commands. I'll concede that its a little awkward that the keyboard commands for cycling the items lives in the defaultmap, but those will be removed by the end of next week anyway.